src/sentry/static/sentry/app/views/settings/components/forms/projectMapperField.tsx

```diff
@@ -252,19 +269,20 @@ export class RenderField extends React.Component<RenderProps, State> {
             )}
           </FieldControlWrapper>
           {nextUrl && (
-            <StyledNextButton
-              type="button"
-              size="small"
-              priority="default"
-              icon={<IconOpen />}
-              href={nextUrl}
-              external
```

Comment:
Double-checking that you wanted to remove `external`.

Reply:
@mgaeta forgot to call that out but yea, this is intentional